event-app-ui/src/components/SignUpForm/SignUpForm.jsx

```diff
@@ -23,10 +24,22 @@ export default function SignUpForm() {
         console.log("!!!", response);
       });
   };
+  const handleSubmit = (e) => {
```

Comment:
nit: validate at each form element rather than form submission something like this
https://www.positronx.io/react-js-input-field-white-empty-spaces-validation-tutorial/